explorer/fuzzing/ast_to_proto_test.cpp

```diff
@@ -124,7 +124,8 @@ TEST(AstToProtoTest, SetsAllProtoFields) {
 auto main(int argc, char** argv) -> int {
   ::testing::InitGoogleTest(&argc, argv);
   // gtest should remove flags, leaving just input files.
-  Carbon::Testing::carbon_files =
-      new std::vector<llvm::StringRef>(&argv[1], &argv[argc]);
+  auto carbon_files =
+      std::make_unique<std::vector<llvm::StringRef>>(&argv[1], &argv[argc]);
+  Carbon::Testing::carbon_files = carbon_files.get();
   return RUN_ALL_TESTS();
```

Comment:
If we're going to leave it dangling after `RUN_ALL_TESTS()` anyway, does this need to live on the heap at all?
```suggestion
  std::vector<llvm::StringRef> carbon_files(&argv[1], &argv[argc]);
  Carbon::Testing::carbon_files = &carbon_files;
  return RUN_ALL_TESTS();
```

Reply:
The root issue is actually bugprone-exception-escape. The suggestion brings it back, digging for other solutions...

Reply:
Per discussion, disabling bugprone-exception-escape because it's unlikely to find real bugs for us, given we avoid exceptions.